cmd/limactl/template.go

```diff
@@ -0,0 +1,137 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/lima-vm/lima/pkg/limatmpl"
+	"github.com/lima-vm/lima/pkg/limayaml"
+	"github.com/lima-vm/lima/pkg/store/dirnames"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newTemplateCommand() *cobra.Command {
+	templateCommand := &cobra.Command{
+		Use:           "template",
+		Aliases:       []string{"tmpl"},
+		Short:         "Lima template management",
+		SilenceUsage:  true,
+		SilenceErrors: true,
+		GroupID:       advancedCommand,
+		// The template command is still hidden because the subcommands and options are still under development
+		// and subject to change at any time.
+		Hidden: true,
+	}
+	templateCommand.AddCommand(
+		newTemplateCopyCommand(),
+		newTemplateValidateCommand(),
+	)
+	return templateCommand
+}
+
+// The validate command exists for backwards compatibility, and because the template command is still hidden.
+func newValidateCommand() *cobra.Command {
+	validateCommand := newTemplateValidateCommand()
+	validateCommand.GroupID = advancedCommand
+	return validateCommand
+}
+
+var templateCopyExample = `  Template locators are local files, file://, https://, or template:// URLs
+
+  # Copy default template to STDOUT
+  limactl template copy template://default -
+
+  # Copy template from web location to local file
+  limactl template copy https://templates.r.us/lima.yaml mighty-machine.yaml
```

Comment:
What is r.us?

Reply:
This should probably use example.com

Reply:
> What is r.us?

It was supposed to be a joke, referencing "Toys'R'Us" (https://www.toysrus.ca) and similar names like "Babies'R'Us" or "Pets'R'Us".

> This should probably use example.com

Yes, the `--help` output should not try to be funny (or confusing). Sorry about that!

Changed to `example.com.

Reply:
Just realized that `mighty-machine.yaml` was also a joke (reference to https://www.youtube.com/channel/UCgCV4LtUjiBL8-6cqqMeuFQ). But I think this one is still ok, or do you want me to change it too?